scene/2d/area_2d.compat.cpp

```diff
@@ -32,17 +32,17 @@
 
 #include "area_2d.h"
 
-void Area2D::set_priority_compat_72749(float p_priority) {
-	set_priority(p_priority);
+static void set_priority_compat_72749(Area2D *p_this, float p_priority) {
+	p_this->set_priority(p_priority);
 }
 
-float Area2D::get_priority_compat_72749() const {
-	return get_priority();
+static float get_priority_compat_72749(Area2D *p_this) {
```

Comment:
This one doesn't seem to work as expected:
```
Validate extension JSON: Error: Hash mismatch for 'classes/Area2D/methods/get_priority'. This means that the function has changed and no compatibility function was provided.
```

Is that because of the removal of `const`?